lib/model/model.go

```diff
@@ -2226,25 +2226,13 @@ func (m *Model) State(folder string) (string, time.Time, error) {
 	return state.String(), changed, err
 }
 
-func (m *Model) PullErrors(folder string, page, perpage int) ([]FileError, error) {
+func (m *Model) PullErrors(folder string) ([]FileError, error) {
 	m.fmut.RLock()
+	defer m.fmut.RUnlock()
 	if err := m.checkFolderRunningLocked(folder); err != nil {
```

Comment:
Just thinking out loud. I suspect after calling this functions we usually reach into the runner, so perhaps this function should return the runner or error (paused/not exists).

Reply:
Sometimes the runner is used afterwards, but also a few other folder map values, and at least once it is used without needing the runner afterwards. It seems more straightforward to me if it is a status check, as opposed to a lookup that also does some extended status checking. And I guess this isn't really relevant performance wise (?).

Reply:
Sure, just checking.